src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,14 @@
+const initialState = {
+  categories: [],
+};
+
+function categoriesReducer(state = initialState, action) {
+  switch (action.type) {
+    case 'CATEGORY':
+      return 'Under Construction';
+    default:
+      return state;
+  }
+}
+
+export default categoriesReducer;
```

Comment:
- [ ] Kindly refactor your categories reducer to use slice as stated in the requirements.